### PR TITLE
Feature/add get current user

### PR DESCRIPTION
**Pull request by s3341458**

### CODE REVIEW

File: typescript/src/merchi.ts [hunk 1]
```diff
@@ -1,66 +1,69 @@
...
+import { Company } from "./entities/company";
+import { ComponentTag } from "./entities/component_tag";
+import { EnrolledDomain } from "./entities/enrolled_domain";
+import { CountryTax } from "./entities/country_tax";
+import { Item } from "./entities/item";
+import { DomainTag } from "./entities/domain_tag";
+import { DraftComment } from "./entities/draft_comment";
+import { Notification } from "./entities/notification";
+import { Payment } from "./entities/payment";
+import { CompanyInvitation } from "./entities/company_invitation";
```
Comment (DanielCollins) [inline typescript/src/merchi.ts]: this has nothing to do with current users.

**Comment by DanielCollins** [comment]
too much unrelated junk here to be able to read it

**Comment by s3341458** [comment]
I am sorry my vim auto fixer mistouched some code, I set it aggressive fix things for for save my typing time.

**Comment by s3341458** [comment]
prefer job profile which include the changes in this pr and other related changes

**Pull request closed without merging**